composer.json

```
@@ -10,6 +10,8 @@
             "email": "eldad@appwrite.io"
         }
     ],
+
```

```suggestion
    "scripts": {
        "check": "./vendor/bin/phpstan analyse --level max src tests"
```
Missing the `scripts` object with the `check` script used by the docker image to run PHPStan.